examples/compose/plot_column_transformer_mixed_types.py

```diff
@@ -44,9 +44,9 @@
 # X = titanic.frame.drop('survived', axis=1)
 # y = titanic.frame['survived']
 
-###############################################################################
+# %%
 # Use ``ColumnTransformer`` by selecting column by names
-###############################################################################
+# %%
```

Comment:
Revert this.

The second `##########################` was used for a header:

This PR:

<img width="858" alt="Screen Shot 2020-05-26 at 6 13 07 PM" src="https://user-images.githubusercontent.com/5402633/82955319-9141ef80-9f7c-11ea-97dc-f79f6501d577.png">

Master: 

<img width="639" alt="Screen Shot 2020-05-26 at 6 13 23 PM" src="https://user-images.githubusercontent.com/5402633/82955345-a159cf00-9f7c-11ea-89b4-c859842b0fe7.png">